stubs/nova_resource_test.blade.php

```diff
@@ -0,0 +1,340 @@
+@php($shouldUseStatus = version_compare(app()->version(), '7', '<'))
+namespace App\Tests;
+
+use App\Models\{{$entity}};
+use Illuminate\Support\Collection;
+use RonasIT\Support\Tests\ModelTestState;
+use RonasIT\Support\Traits\AuthTestTrait;
+@if($shouldUseStatus)
+use Symfony\Component\HttpFoundation\Response;
+@endif
+
+class {{$entity}}Test extends TestCase
+{
+    use AuthTestTrait;
+
+    protected $user;
+
+    public function setUp() : void
+    {
+        parent::setUp();
+
+        $this->user = 1;
+
+        $this->skipDocumentationCollecting();
+    }
+
+    public function testCreate(): void
+    {
+        $modelState = new ModelTestState({{$entity}}::class);
```

Comment:
let's move it to the `setUp` method and use the next syntax

```suggestion
        self::${{$entity}}State ??= new ModelTestState({{$entity}}::class);
```